<APP_NAME>/migrations/0001_initial.py

```
@@ -0,0 +1,62 @@
+                ("period_end", models.DateTimeField()),
+                ("anomalies", models.JSONField(blank=True, default=list)),
+                ("summary_text", models.TextField()),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                (
+                    "service",
+                    models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name="digests", to="<APP_NAME>.service"),
+                ),
+            ],
+            options={"ordering": ["-created_at"]},
+        ),
+    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)